src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -62,7 +62,7 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
                 '<div class="modes"></div>',
                 '<h3>{{ d.name }}</h3>',
                 '<h5 class="distance-minutes"></h5>',
-                '<img src="{{#if d.wide_image}}{{ d.wide_image }}{{^}}https://placehold.it/300x150{{/if}}" width="300px" height="150px" />',
+                '<img src="{{#if d.wide_image}}{{ d.wide_image }}{{^}}http://placehold.it/300x150{{/if}}" />',
```

Comment:
Good call. We need all https from now on or else we'll get mixed-content warnings.